<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1547</width>
    <height>1068</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MeshTools</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>81</width>
      <height>981</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="RegionSelectButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Select Region</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/new/icons/region-select.ico</normaloff>
       <normalon>:/new/icons/region-select.ico</normalon>
       <disabledoff>:/new/icons/region-select.ico</disabledoff>
       <disabledon>:/new/icons/region-select.ico</disabledon>
       <activeoff>:/new/icons/region-select.ico</activeoff>
       <activeon>:/new/icons/region-select.ico</activeon>
       <selectedoff>:/new/icons/region-select.ico</selectedoff>
       <selectedon>:/new/icons/region-select.ico</selectedon>:/new/icons/region-select.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="ExportMesh">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Export Fbx Meshes</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/new/icons/mesh-export.ico</normaloff>:/new/icons/mesh-export.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="ImportDumpData">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Import Usgs Data</string>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/new/icons/dump-import.ico</normaloff>
       <normalon>:/new/icons/dump-import.ico</normalon>
       <disabledoff>:/new/icons/dump-import.ico</disabledoff>
       <disabledon>:/new/icons/dump-import.ico</disabledon>
       <activeoff>:/new/icons/dump-import.ico</activeoff>
       <activeon>:/new/icons/dump-import.ico</activeon>
       <selectedoff>:/new/icons/dump-import.ico</selectedoff>
       <selectedon>:/new/icons/dump-import.ico</selectedon>:/new/icons/dump-import.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="GoogleEarthDump">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Import Google Earth Dump</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/new/icons/google-earth-dump.ico</normaloff>
       <normalon>:/new/icons/google-earth-dump.ico</normalon>
       <disabledoff>:/new/icons/google-earth-dump.ico</disabledoff>
       <disabledon>:/new/icons/google-earth-dump.ico</disabledon>
       <activeoff>:/new/icons/google-earth-dump.ico</activeoff>
       <activeon>:/new/icons/google-earth-dump.ico</activeon>
       <selectedoff>:/new/icons/google-earth-dump.ico</selectedoff>
       <selectedon>:/new/icons/google-earth-dump.ico</selectedon>:/new/icons/google-earth-dump.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="ReferencePoint">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Reference Gps Coordinate</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/new/icons/gps_reference.ico</normaloff>:/new/icons/gps_reference.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="ImportFbxData">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Import Fbx File</string>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/new/icons/fbx_import.ico</normaloff>
       <normalon>:/new/icons/fbx_import.ico</normalon>
       <disabledoff>:/new/icons/fbx_import.ico</disabledoff>
       <disabledon>:/new/icons/fbx_import.ico</disabledon>
       <activeoff>:/new/icons/fbx_import.ico</activeoff>
       <activeon>:/new/icons/fbx_import.ico</activeon>
       <selectedoff>:/new/icons/fbx_import.ico</selectedoff>
       <selectedon>:/new/icons/fbx_import.ico</selectedon>:/new/icons/fbx_import.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="ImportKml">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Import Kml File</string>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/new/icons/kml_import.ico</normaloff>
       <normalon>:/new/icons/kml_import.ico</normalon>
       <disabledoff>:/new/icons/kml_import.ico</disabledoff>
       <disabledon>:/new/icons/kml_import.ico</disabledon>
       <activeoff>:/new/icons/kml_import.ico</activeoff>
       <activeon>:/new/icons/kml_import.ico</activeon>
       <selectedoff>:/new/icons/kml_import.ico</selectedoff>
       <selectedon>:/new/icons/kml_import.ico</selectedon>:/new/icons/kml_import.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="OGLWidget" name="widget_ogl" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>1471</width>
      <height>921</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>1</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reference Gps Coordinate:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Longitude:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Latitude:</string>
    </property>
   </widget>
   <widget class="QLabel" name="longitudeValue">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="latitudeValue">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Scissor Rectangle:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Longitude Start:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>50</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Latitude Start:</string>
    </property>
   </widget>
   <widget class="QLabel" name="longitudeStart">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="longitudeEnd">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="latitudeStart">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="latitudeEnd">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>381</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>771</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="loadSaveProgressBar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>521</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1547</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OGLWidget</class>
   <extends>QWidget</extends>
   <header>oglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
